### ISSUE TITLE
MOL2 format visualization

### ISSUE THREAD

**Issue by stefdoerr** [molstar/molstar#178]
Hi I have an issue with molstar visualizing MOL2 files. 
If I pass it a water molecule it's missing a bond which is correctly defined in the file

![image](https://user-images.githubusercontent.com/7935362/117405896-ffc15880-af14-11eb-8cd2-45027ac53083.png)

```
@<TRIPOS>MOLECULE
    HOH
    3     2     0     0     0
SMALL
USER_CHARGES


@<TRIPOS>ATOM
      1 O           0.0020    0.3832    0.0000 oh         1 HOH     -0.785000
      2 H          -0.7702   -0.1875    0.0000 ho         0 HOH      0.392000
      3 H0          0.7682   -0.1957    0.0000 ho         0 HOH      0.392000
@<TRIPOS>BOND
     1    1    2 1
     2    1    3 1
```

Also as an extra question. While it handles single, double and triple bonds correct it doesn't visualize unkekulized aromatic rings. Is there any option to enable that?

![image](https://user-images.githubusercontent.com/7935362/117406385-a73e8b00-af15-11eb-91da-6c2c0d3431cb.png)
vs PyMOL
![image](https://user-images.githubusercontent.com/7935362/117406692-1320f380-af16-11eb-81b9-e649b27b549d.png)

```
@<TRIPOS>MOLECULE
    UNL
   30    32     0     0     0
SMALL
USER_CHARGES


@<TRIPOS>ATOM
      1 C           3.2867   -0.7761    0.4714 ca         1 UNL     -0.172500
      2 C0          4.6360   -0.6536    0.1004 ca         1 UNL     -0.104500
      3 C1          5.0354    0.3643   -0.7794 ca         1 UNL     -0.161000
      4 C2          4.0828    1.2570   -1.2941 ca         1 UNL     -0.104500
      5 C3          2.7334    1.1351   -0.9239 ca         1 UNL     -0.172500
      6 C4          2.3272    0.1277   -0.0305 ca         1 UNL      0.074600
      7 N           0.9486    0.0266    0.3058 ns         1 UNL     -0.469100
      8 C5          0.3788   -0.4232    1.4857 za         1 UNL      0.715300
      9 O           1.0370   -0.7522    2.4571 zb         1 UNL     -0.606100
     10 C6         -1.1051   -0.4483    1.4824 cc         1 UNL     -0.230400
     11 C7         -2.0636   -0.1265    0.4348 ca         1 UNL     -0.062800
     12 C8         -1.9230    0.2730   -0.9081 ca         1 UNL     -0.100000
     13 C9         -3.0946    0.4942   -1.6627 ca         1 UNL     -0.142000
     14 C10        -4.3706    0.3116   -1.0830 ca         1 UNL     -0.116000
     15 C11        -4.4861   -0.0983    0.2589 ca         1 UNL     -0.134000
     16 C12        -3.3043   -0.3058    0.9772 ca         1 UNL     -0.057200
     17 N0         -3.2171   -0.7217    2.2848 na         1 UNL     -0.184400
     18 C13        -1.8492   -0.8195    2.6039 cd         1 UNL     -0.020100
     19 H           2.9953   -1.5841    1.1350 ha         1 UNL      0.149500
     20 H0          5.3706   -1.3516    0.4919 ha         1 UNL      0.130500
     21 H1          6.0792    0.4573   -1.0657 ha         1 UNL      0.131000
     22 H2          4.3891    2.0422   -1.9795 ha         1 UNL      0.130500
     23 H3          2.0020    1.8300   -1.3265 ha         1 UNL      0.149500
     24 H4          0.3119    0.3522   -0.4065 hn         1 UNL      0.320500
     25 H5         -0.9564    0.4077   -1.3782 ha         1 UNL      0.136000
     26 H6         -3.0155    0.8041   -2.7009 ha         1 UNL      0.134000
     27 H7         -5.2648    0.4826   -1.6757 ha         1 UNL      0.134000
     28 H8         -5.4583   -0.2507    0.7186 ha         1 UNL      0.136000
     29 H9         -3.9917   -0.9237    2.9012 hn         1 UNL      0.305700
     30 H10        -1.5135   -1.1305    3.5853 h4         1 UNL      0.188000
@<TRIPOS>BOND
     1    1    2 ar
     2    1    6 ar
     3    1   19 1
     4    2    3 ar
     5    2   20 1
     6    3    4 ar
     7    3   21 1
     8    4    5 ar
     9    4   22 1
    10    5    6 ar
    11    5   23 1
    12    6    7 1
    13    7    8 am
    14    7   24 1
    15    8    9 2
    16    8   10 1
    17   10   11 ar
    18   10   18 ar
    19   11   12 ar
    20   11   16 ar
    21   12   13 ar
    22   12   25 1
    23   13   14 ar
    24   13   26 1
    25   14   15 ar
    26   14   27 1
    27   15   16 ar
    28   15   28 1
    29   16   17 ar
    30   17   18 ar
    31   17   29 1
    32   18   30 1
```

**Comment by dsehnal** [comment]
Hi,

the bug is caused by Mol* sorting the input atoms by residue number internally but not adjusting the indices in the bond list. Changing the H residue number from 0 to 1 or 2 will fix the issue. I will look into fixing this soon.

-----------

As for the ring visuals: @arose any thoughts on this?

**Comment by arose** [comment]
> As for the ring visuals: @arose any thoughts on this?

Just updated the issue we have on it #1

**Comment by dsehnal** [comment]
Should be fixed in 881d4d2a99b5daea149807a5f035b90d14b2aff2.

**Comment by arose** [comment]
Hi @stefdoerr, see #211, there is no an option to show aromatic rings. Cheers
